src/test/java/com/amazon/opendistroforelasticsearch/sql/antlr/visitor/AntlrSqlParseTreeVisitorTest.java

```diff
@@ -74,6 +74,42 @@ public void selectStarShouldReturnEmptyProductAsQueryVisitingResult() {
         Assert.assertTrue(result.isCompatible(new Product(emptyList())));
     }
 
+    @Test(expected = SqlFeatureNotImplementedException.class)
```

Comment:
Could you also add the exception message assert?

Reply:
Will do, thanks!